src/components/IndividualArticle.jsx

```diff
@@ -2,10 +2,12 @@ import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
 import { Link } from "react-router-dom";
 import { getArticleById } from "../../api";
+import ArticleComments from "./ArticleComments";
 
 export default function IndividualArticle() {
   const { article_id } = useParams();
   const [singleArticle, setSingleArticle] = useState({});
+  const [comments, setComments] = useState([]);
```

Comment:
This state should be in Comments component. There's no need for it to be in this one